### PR TITLE
Gradient Descent from scratch issue #2957

### PR DESCRIPTION
**Pull request by Anindyadeep**
<hr>

## Gradient Descent from scratch issue #2957 

issue #2957
Please review my ipynb files, and let me know the changes I have to make, I am feeling some issues of images, as its not showing after adding in the Assets folder.

<hr>

## Domain of Contribution 📊
Machine Learning issue #2957  ML_2_20_Implement_Gradient_Descent_without_using_any_standard_ML_library_like scikit-learn_or_more_(D).ipynb

### CODE REVIEW

**Comment by abhisheks008** [comment]
@Anindyadeep Create a fresh PR in the Machine Learning branch, not in the Main branch. The notebook is good, just add the files in the machine learning branch.
